spec/services/order_details/notices_service_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe OrderDetails::NoticesService do
+  let(:order_detail) { OrderDetail.new }
+  let(:instance) { described_class.new(order_detail) }
+
+  describe "statuses" do
+    subject { instance.notices }
+
+    it "shows nothing for a blank order detail" do
+      is_expected.to be_blank
+    end
+
+    it "shows nothing for a canceled order detail" do
+      allow(order_detail).to receive(:in_review?).and_return(true)
+      allow(order_detail).to receive(:canceled?).and_return(true)
+
+      is_expected.to be_blank
+    end
+
+    it "shows in review if the order is in review" do
+      allow(order_detail).to receive(:in_review?).and_return(true)
+
+      is_expected.to eq([:in_review])
+    end
+
+    it "shows in dispute" do
+      allow(order_detail).to receive(:in_dispute?).and_return(true)
+
+      is_expected.to eq([:in_dispute])
+    end
+
+    it "shows in dispute" do
```

Comment:
You have a duplicated test name here. I think you should call this one "shows global admin must resolve"

Reply:
good catch!